polyfill/lib/ecmascript.mjs

```diff
@@ -1910,7 +1910,7 @@ export const ES = ObjectAssign({}, ES2022, {
       return temporalTimeZoneLike;
     }
     const identifier = ES.ToString(temporalTimeZoneLike);
-    return ES.ParseTemporalTimeZone(identifier);
+    return ES.GetCanonicalTimeZoneIdentifier(identifier);
```

Comment:
A use case that's come up a few times in various issues filed by users has been extracting the date or time units from a timestamp string that has an offset but does not have a time zone annotation. For example, given a string like `'2020-01-01T00:00-08:00'` what's the date shown in that string?

Currently the pattern for this use case is nicely ergonomic: 

```js
function getLocalDateOfTimestamp(s) {
  return Temporal.Instant.from(s).toZonedDateTimeISO(s).toPlainDate();
}
getLocalDateOfTimestamp('2020-01-01T00:00-08:00');
// => 2020-01-01
```

Does this change break this pattern?

Reply:
As per today's meeting, we'll discuss this as part of #2104.